pandas/core/groupby/groupby.py

```diff
@@ -1572,6 +1572,13 @@ def apply(self, func, *args, include_groups: bool = True, **kwargs) -> NDFrameT:
         behavior or errors and are not supported. See :ref:`gotchas.udf-mutation`
         for more details.
 
+        Groups for which ``func`` returns ``None`` will be filtered from the result.
+
+        .. versionchanged:: 2.2.2
+
+            In case all groups are filtered from the result, an empty DataFrame
+            with the columns and dtypes of the original dataframe will be returned.
+
```

Comment:
```suggestion
```

(the example below is sufficient)

Reply:
Removed the description, only left the example